pnpm-lock.yaml

```diff
@@ -348,6 +348,7 @@ importers:
       react-icons: ^4.3.1
       react-transition-group: ^4.4.2
       react-use: ^17.2.1
+      react-youtube: ^7.13.1
```

Comment:
I've seen [lite-youtube-embed](https://github.com/paulirish/lite-youtube-embed) which claimed to render 224x faster while giving the same functionalities. Let's try it!

Reply:
How would I load this into the front-end project? I aready `pnpm install`ed it, but I am not sure how to load those files. The NPM package description says to simply treat it like a regular JS file, but I am not sure if I can do that here. @DukeManh

Reply:
I figured it out! I had to add a few `import` lines in the `_app.tsx` file. 😌

Reply:
Actually, nevermind, it seemed to work once and when I restarted the program it stopped working...

Reply:
You use `pnpm add package-name` to add a package, and `pnpm install` to install dependencies and update the lock file.  With `npm` you use `install` for both meanings.

Reply:
So, I was searching for a way to make this work, and I found this issue: vercel/next.js#11619. It seems that we are supposed to directly load the file with `import`, so that the `HTMLElement is not defined` error does not happen. I am not sure if there is a better way to make this work.

Reply:
What if you disable SSR and load it dynamically https://nextjs.org/docs/advanced-features/dynamic-import#with-no-ssr.  I've had to do this in the past with other similar components.

Reply:
@dbelokon they have a React ports too, have you tried that?

Reply:
https://github.com/ibrahimcesar/react-lite-youtube-embed

Reply:
I'm going to try using the React port. I didn't know they had one

Reply:
The React port project worked great!